README.md

```diff
@@ -94,6 +94,10 @@ that full support will remain only for `json` and  `yaml`.
 
 Users needing to import XML OSCAL artifacts are recommended to look at NIST's XML to json conversion page [here](https://github.com/usnistgov/OSCAL/tree/master/json#oscal-xml-to-json-converters).
 
+## Tutorials
+
+List of tutorials [here](docs/tutorials/tutorials.md).
+
```

Comment:
Link the tutorials to the website (ibm.github.io/compliance-trestle/tutorials/tutorials.html) (or similar rather than on the github page.

Reply:
Fixed...I think!?